spark/datadog_checks/spark/spark.py

```diff
@@ -495,16 +493,22 @@ def _spark_job_metrics(self, instance, running_apps, addl_tags, requests_config)
                 SPARK_APPS_PATH,
                 SPARK_SERVICE_CHECK, requests_config, addl_tags, app_id, 'jobs')
 
+            tags = ['app_name:{}'.format(app_name)]
+            tags.extend(addl_tags)
+            status_cache = {}
+
             for job in response:
 
                 status = job.get('status')
+                if status not in status_cache:
+                    status_cache[status] = {'count': 0, 'tags': ['status:{}'.format(str(status).lower())]}
+                    status_cache[status]['tags'].extend(tags)
 
-                tags = ['app_name:%s' % str(app_name)]
-                tags.extend(addl_tags)
-                tags.append('status:%s' % str(status).lower())
+                status_cache[status]['count'] += 1
+                self._set_metrics_from_json(status_cache[status]['tags'], job, SPARK_JOB_METRICS)
 
-                self._set_metrics_from_json(tags, job, SPARK_JOB_METRICS)
-                self._set_metric('spark.job.count', INCREMENT, 1, tags)
+            for data in itervalues(status_cache):
+                self._set_metric('spark.job.count', GAUGE, data['count'], data['tags'])
```

Comment:
metric is named `count`, why not use COUNT ? Additionally, this would also spare us the `status_cache` logic to keep track of the actual value, as the `count` method will do it for us.

Reply:
Done!